wfdb/plot/plot.py

```diff
@@ -210,21 +217,23 @@ def create_figure(n_subplots, figsize):
         The number of subplots to generate.
     figsize : tuple
         The figure's width, height in inches. 
+	sharex, sharey : bool or {'none', 'all', 'row', 'col'}, default: False
+		Controls sharing of properties among x (`sharex`) or y (`sharey`) axes:
+			True or 'all': x- or y-axis will be shared among all subplots.
+			False or 'none': each subplot x- or y-axis will be independent.
+			'row': each subplot row will share an x- or y-axis.
+			'col': each subplot column will share an x- or y-axis.
 
     Returns
     -------
     fig : matplotlib plot object
         The entire figure that will hold each subplot.
     axes : list
         The information needed for each subplot.
-
     """
-    fig = plt.figure(figsize=figsize)
-    axes = []
-
-    for i in range(n_subplots):
-        axes.append(fig.add_subplot(n_subplots, 1, i+1))
-
+	fig, axes = plt.subplots(
+		nrows=n_subplots, ncols=1, sharex=sharex, sharey=sharey, figsize=figsize
```

Comment:
Something is wrong here I think.. a tab instead of spaces? It's causing the tests to fail

Reply:
this is what i get for editing code directly on github instead of locally cloning the repo 😁